File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):
    arr = []
    answer = 0
    for i in range(len(t)-len(p)+1):
        arr.append(t[i:i+len(p)])
    for j in arr:
        if int(j) <= int(p):
            answer += 1
    return answer

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    let arr = [];
    for(let i=0; i<=t.length - p.length;i++){
        arr.push(t.substring(i,i+p.length));
    }
    let answer = 0;
    
    for(let j=0; j<arr.length;j++){
        if(parseInt(p) >= parseInt(arr[j])){
            answer++;
        }
    }
    return answer;
}